src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -366,3 +368,30 @@ def drexel_fast_zonal(geojson, key):
     result = {int(k): v for k, v in res.json()[key].items()}
 
     return result
+
+
+def wkaoi_from_huc(huc):
+    huc_len = len(huc)
+    if huc_len not in [8, 10, 12]:
+        raise ValidationError(f'Specified HUC {huc} is {huc_len} digits. '
+                              'Must be 8, 10, or 12.')
+
+    if huc_len == 8:
+        huc_col = 'huc08'
```

Comment:
This is because in some places (like the WKAoI `huc8__9`) we use `huc8`, and in others (like the table `boundary_huc08` and the column name `huc08`) we use `huc08`.